CorsixTH/Lua/room.lua

```diff
@@ -429,10 +429,24 @@ function Room:createDealtWithPatientCallback(humanoid)
 end
 
 --! Get the current staff member.
--- Can be overridden in rooms with multiple staff members to return the desired one.
+-- In multi-occupancy rooms this returns the staff member with the minimum service quality
 --!return (staff) The current staff member.
 function Room:getStaffMember()
-  return self.staff_member
+  if not self.staff_member_set then return self.staff_member end
```

Comment:
The simpler system is to always have a set of staff members, even for rooms with 1 staff member. It allows to always use the same system to access staff members for every room, instead of having to distinguish for different cases all throughout the code.